### PR TITLE
Update to net 6.0 

### PR DESCRIPTION
**Pull request by pkoelbl**
The purpose of this is to update to the current LTS for .net(6.0). 

It is also to enable arm64 support for M1 Macs.

### CODE REVIEW

**Comment by elenavillamil** [comment]
Hi,

Thank you for your PR. Could you add more details describing background and need for this PR? We have tested bridge on Mac M1 and arm64 and it worked fine. This is a big PR with high risk. It will also require pipeline updates and heavy testing. We will only consider it if there is a solid reason / justification for the change.

 Thank you.

**Comment by pkoelbl** [comment]
On an M1 Mac using B2K to connect to Docker-Desktop with k8s enabled running on the same M1 doesn't work. I get this error:
`2022-09-13T20:30:08.269Z | Common Extension Root    | ERROR | Error: binaries-utility-ensure-cli-error <json>{"isUsingLocalBinaries":"false","remoteName":"dev-container"}</json> <stack>Error: qemu-x86_64: Could not open '/lib64/ld-linux-x86-64.so.2': No such file or directory\n\n  at ChildProcess.<anonymous> (/root/.vscode-server/extensions/mindaro.mindaro-1.0.120220817/dist/extension.js:3:236414)\n  at ChildProcess.emit (node:events:526:28)\n at Process.ChildProcess._handle.onexit (node:internal/child_process:291:12)</stack>`

File: src/routingmanager/Properties/launchSettings.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (hsubramanianaks) [inline src/routingmanager/Properties/launchSettings.json:1]: Thank you for the PR, Could I know why this new file was introduced?

**Review by hsubramanianaks — COMMENTED** [review]
/AzurePipelines run

**Review by hsubramanianaks — COMMENTED** [review]
Thanks for your PR @pkoelbl , while testing this we found issues 

-  EndpointManagerLauncher failed in Windows
- While disconnecting fails with connections refused with wrong port number in WSL/Linux. 

we are looking into these issues.

**Comment by hsubramanianaks** [comment]
Closing this branch as it is taken care of this [PR](https://github.com/Azure/Bridge-To-Kubernetes/pull/108). Thanks @pkoelbl for contribution. Appreciate your work.

**Pull request closed without merging**